### PR TITLE
Update docs, apply feedback

### LINKED ISSUE #157: export/import of the raw credential internals
from discussion in matrix... tldr if you're holding a `SecurityKey` or similar, being able to get at the underlying `Credential` fields somehow.

### PR DESCRIPTION
**Pull request by Firstyear**
Fixes #157

- [ x ] cargo fmt has been run
- [ x ] cargo test has been run and passes
- [ ] documentation has been updated with relevant examples (if relevant)

### CODE REVIEW

**Comment by Firstyear** [comment]
@devsnek

File: webauthn-rs/src/lib.rs [hunk 1]
```diff
@@ -775,7 +836,11 @@ impl Webauthn {
         let extensions = None;
         let credential_algorithms = self.algorithms.clone();
         let require_resident_key = false;
-        let policy = Some(UserVerificationPolicy::Preferred);
+        let policy = if cfg!(feature = "danger-user-presence-only-security-keys") {
+            Some(UserVerificationPolicy::Discouraged_DO_NOT_USE)
```
Comment (devsnek) [inline webauthn-rs/src/lib.rs:840]: iirc this will hit a `warn!()` in the core library, maybe guard that warning on this feature not being enabled?

**Review by devsnek — COMMENTED** [review]
overall seems great

File: webauthn-rs/src/lib.rs (hunk 1 above)
Comment (Firstyear) [reply webauthn-rs/src/lib.rs:840]: The warning in core probably isn't needed at this point, so I'll remove it :)

**Comment by Firstyear** [comment]
@devsnek Done, warning removed.

**Pull request merged**